src/main/java/org/broadinstitute/hellbender/tools/walkers/varianteval/evaluators/VariantSummary.java

```diff
@@ -167,9 +167,9 @@ private boolean overlapsKnownCNV(VariantContext cnv, VariantEvalContext context)
         if ( knownCNVs != null ) {
             List<Feature> overlaps = context.queryFeaturesIncludingOverlapping(knownCNVs, new SimpleInterval(cnv.getContig(), cnv.getStart(), cnv.getEnd()));
             GenomeLocParser parser = new GenomeLocParser(context.getSequenceDictionaryForDrivingVariants());
-            GenomeLoc loc1 = parser.createGenomeLoc(cnv);
+            GenomeLoc loc1 = parser.createGenomeLoc(cnv, true);
             for (Feature vc : overlaps) {
-                GenomeLoc loc2 = parser.createGenomeLoc(vc);
+                GenomeLoc loc2 = parser.createGenomeLoc(vc, true);
```

Comment:
Add a comment above these lines pointing out that we're explicitly allowing intervals that go off the end of the reference